app/views/shared/_alerts.html.erb

```diff
@@ -4,6 +4,9 @@
       <div class="alert alert-<%= klass %>">
         <button type="button" class="close" data-dismiss="alert">&times;</button>
         <%= message %>
+        <% if display_access_request_link?(type) %>
+          <%= link_to_request_access %>
+        <% end %>
```

Comment:
Yeah, I just don't like conditionals in templates as well. But I can't find a better solution. Leave it as it is.